app/views/shared/dialogs/_dialog_provision.html.haml

```diff
@@ -50,4 +50,5 @@
                 :finish_submit_endpoint => finish_submit_endpoint,
                 :api_submit_endpoint    => api_submit_endpoint,
                 :api_action             => api_action,
-                :cancel_endpoint        => cancel_endpoint}
+                :cancel_endpoint        => cancel_endpoint,
+                :open_url               => open_url}
```

Comment:
@ZitaNemeckova is causing an error on upstream, please take a look
![image](https://user-images.githubusercontent.com/3450808/44481180-27601d80-a613-11e8-88bb-2679281df88f.png)

Reply:
Fixed in https://github.com/ManageIQ/manageiq-ui-classic/pull/4521